modules/ROOT/pages/replace-license-key.adoc

```diff
@@ -13,42 +13,47 @@ this node is named `rtf-controller-1` by default.
 
 The `rtfctl` utility is required to replace your Mule license key on Runtime Fabric. Follow the steps in xref:install-rtfctl.adoc[Install rtfctl] before continuing.
 
+For Runtime Fabric on VMs / Bare Metal, the `rtfctl` binary is in directory `/opt/anypoint/runtimefabric/`.
+
```

Comment:
@kmendiratta-crm Can you confirm that this is the correct additions you asked me to add?